integration-tests/tests/common/mod.rs

```diff
@@ -76,6 +82,8 @@ impl IntegrationTest {
             .arg(self.code)
             .arg("--shim")
             .arg(self.shim)
+            .stdin(Stdio::inherit())
+            .stdout(Stdio::piped())
```

Comment:
Based on the comments in the [read PR](https://github.com/enarx/enarx/pull/711), I think these should be `piped()` and there should be a `.stderr`.

Reply:
Thanks for letting me know :)